Additional information, fixed typos and examples. (#44)
DataDog/serilog-sinks-datadog-logs: README.md

    .WriteTo.DatadogLogs("<API_KEY>")
    .CreateLogger();
```
## Configuration

By default the logs are forwarded to Datadog via **HTTPS** on port 443 to the US site: `https://http-intake.logs.datadoghq.com)`.  

> You can change the site to EU by using the `Url` property on the `DatadogConfiguration` object and set it to `https://http-intake.logs.datadoghq.eu`. This can be done by passing in the url in the constructor, or by setting the property directly.

Reviewer: ```suggestion
You can change the site to EU by using the `Url` property on the `DatadogConfiguration` object and set it to `https://http-intake.logs.datadoghq.eu`. This can be done by passing in the URL in the constructor, or by setting the property directly.
```

    .CreateLogger();
```

## Output

Sending the following log:

```csharp
var log = new LoggerConfiguration()
    .WriteTo.DatadogLogs("<API_KEY>")
    .CreateLogger();
    
var position = new { Latitude = 25, Longitude = 134 };
var elapsedMs = 34;

log.Information("Processed {@Position} in {Elapsed:000} ms.", position, elapsedMs);
```

> Results in the following JSON Object log entry when viewed in DataDog:

Reviewer: ```suggestion
Results in the following JSON Object log entry when viewed in Datadog:
```

}
```

> OBS!  

Reviewer: ```suggestion 
```

> If a log message exceeds the max size allowed (defined as: `_maxSize = 2 * 1024 * 1024 - 51;`) the message is **not sent** to Datadog.

Reviewer 2: ```suggestion
If a log message exceeds the max size allowed (defined as: `_maxMessageSize = 256 * 1024;`) the message is **not sent** to Datadog.
```

> If a log message exceeds the max size allowed (defined as: `_maxSize = 2 * 1024 * 1024 - 51;`) the message is **not sent** to Datadog.

Reviewer 2: `_maxSize` is the maximum size of the payload that contains several logs.

}
```

## Inner workings

> The logger asynchronously emits events in batches that are set to run every 2 seconds. 10 retries are made, spaced out by an exponential retry using the following formula:  

Reviewer: ```suggestion
The logger asynchronously emits events in batches that are set to run every two seconds. Ten retries are made, spaced out by an exponential retry using the following formula:  
```

`int backoff = (int)Math.Min(Math.Pow(2, retry), MaxBackoff);`  

`MaxBackOff` is set to 30. 

> If the batch fails due to an exception (for example invalid url) the exception is swallowed and it keeps trying until there are no retries left. If none of the attempts have gone through it will log to the SelfLog (seriLog feature) using `SelfLog.WriteLine`. 

Reviewer: ```suggestion
If the batch fails due to an exception (for example invalid URL) the exception is swallowed, and it keeps trying until there are no retries left. If none of the attempts have gone through, it logs to the SelfLog (seriLog feature) using `SelfLog.WriteLine`. 
```

Call SelfLog.Enable() at program startup to enable the log.  

Example using Debug.WriteLine:

```
// assume logger setup from earlier example

Serilog.Debugging.SelfLog.Enable(msg => Debug.WriteLine(msg));

log.Information("New ping");
// output

2020-06-30T18:56:07.6409072Z Could not send payload to Datadog: [{"Timestamp":"2020-06-30T20:56:07.2296983+02:00","level":"Information","MessageTemplate":"New ping","message":"New ping","ddsource":"csharp","service":"myService"}]
```
The exception is not logged.

#### Important note 

> due to the periodic batching and exponential backoff retries *an entry might get lost if the application closes or restarts before the batch has been emitted, or retries attempts have not been exhausted.*  

Reviewer 2: ```suggestion
Due to the periodic batching and exponential backoff retries *an entry might get lost if the application closes or restarts before the batch has been emitted, or retries attempts have not been exhausted.*  
```

> If you want to test the logging in a console application make sure to add a delay that allows enough time to process the request. This can be done by using , for example, Thread.Sleep(time) or Task.Delay(time). Blocking the thread by using Thread.Sleep is generally not recommended, but can be used if you just want to verify that the configuration works.

Reviewer: ```suggestion
If you want to test the logging in a console application, make sure to add a delay that allows enough time to process the request. This can be done by using, for example, Thread.Sleep(time) or Task.Delay(time). Blocking the thread by using Thread.Sleep is generally not recommended, but can be used if you just want to verify that the configuration works.
```
